clients/cypress-e2e/cypress/e2e/smoke_test.cy.ts

```diff
@@ -12,16 +12,49 @@ describe("Smoke test", () => {
       "patchRequest"
     );
     cy.intercept("GET", `${API_URL}/privacy-request*`).as("getRequests");
+    cy.intercept("POST", `${API_URL}/privacy-request`).as("postPrivacyRequest");
 
     // Submit the access request from the privacy center
     cy.visit(PRIVACY_CENTER_URL);
     cy.getByTestId("card").contains("Access your data").click();
     cy.getByTestId("privacy-request-form").within(() => {
       cy.get("input#email").type("jenny@example.com");
       cy.get("input#first_name").type("Jenny");
+      cy.get("input#color").clear().type("blue");
       cy.get("button").contains("Continue").click();
     });
 
+    // Verify the privacy request payload
+    cy.wait("@postPrivacyRequest").then((interception) => {
+      expect(interception.request.body).to.deep.equal([
+        {
+          identity: {
+            email: "jenny@example.com",
+            phone_number: "",
+          },
+          custom_privacy_request_fields: {
+            first_name: {
+              label: "First name",
+              value: "Jenny",
+            },
+            last_name: {
+              label: "Last name",
+              value: "",
+            },
+            color: {
+              label: "Color",
+              value: "blue",
+            },
+            tenant_id: {
+              label: "Tenant ID",
+              value: "123",
+            },
```

Comment:
Verifying a few scenarios with this check

- A non-required field still contains a value, even if it's an empty string
- A field with a `default_value` can be overwritten in the form
- A hidden field with a `default_value` is still included with the entered form values in the request